src/libs/ReportActionsUtils.ts

```diff
@@ -117,6 +130,17 @@ function isWhisperAction(reportAction: OnyxEntry<ReportAction>): boolean {
     return (reportAction?.whisperedToAccountIDs ?? []).length > 0;
 }
 
+function isWhisperActionTargetedToOthers(reportAction: OnyxEntry<ReportAction>): boolean {
+    const whisperedToAccountIDs = reportAction?.whisperedToAccountIDs ?? [];
+    if (!whisperedToAccountIDs || whisperedToAccountIDs.length === 0) {
```

Comment:
NAB, but I think this could be made slightly clearer by creating a separate `isWhisper` function and calling it here:

```ts
function isWhisper(reportAction: OnyxEntry<ReportAction>): boolean {
    return (reportAction?.whisperedToAcccountIDs ?? []).length > 0;
}

function isWhisperAcctionTargetedToOthers(reportAction: OnyxEntry<ReportAction>): boolean {
    if (!isWhisper(reportAction)) {
        return false;
    }
    if (!currentUserAccountID) {
        return true;
    }
    return (reportAction?.whisperedToAccountIDs ?? []).includes(currentUserAccountID);
}
```

Reply:
I see that we actually already have such a function right before the one that I created 🤕 . I'll use it.